CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [unreleased]
  ### Added
+- Sub-folders are retained in the response upon updating Folders. ([#1107](https://github.com/ehrbase/ehrbase/pull/1108))
```

Comment:
please also update the link from [#1107]  to [#1108]

Reply:
Done! Would I need to implement tests, or is this feature small enough as is?